nodebuilder/p2p/network.go

```diff
@@ -68,16 +69,30 @@ var networkAliases = map[string]Network{
 	"private": Private,
 }
 
-// listProvidedNetworks provides a string listing all known long-standing networks for things like
-// command hints.
-func listProvidedNetworks() string {
-	var networks string
-	for net := range networksList {
-		// "private" network isn't really a choosable option, so skip
-		if net != Private {
-			networks += string(net) + ", "
-		}
-	}
-	// chop off trailing ", "
-	return networks[:len(networks)-2]
+// orderedNetworks is a list of all known networks in order of priority.
+var orderedNetworks = []Network{Mainnet, Mocha, Arabica, Private}
+
+// GetOrderedNetworks provides a list of all known networks in order of priority.
+func GetOrderedNetworks() []Network {
+	return append([]Network(nil), orderedNetworks...)
+}
+
+// listAvailableNetworks provides a string listing all known long-standing networks for things like CLI hints.
+func listAvailableNetworks() string {
+    var networks []string
+    for _, net := range orderedNetworks {
+		// "private" networks are configured via env vars, so skip
```

Comment:
Nit: remove extra tab for nice alignment

Reply:
think it was a spaces vs tabs thing.. thx